scripts/new_project.kts

```diff
@@ -62,6 +71,7 @@ object NewProject {
         val argumentError = "ERROR: Invalid argument name: Ensure define arguments => app-name={\"MyProject\"} or {\"My Project\"} package-name={com.sample.myproject}"
         when (args.size) {
             1 -> when {
+                args.first() == KEY_HELP -> showMessage(message = HELP_MESSAGE, exitAfterMessage = true)
```

Comment:
What do you think about this suggestion, would it be better? 
```
private fun handleArguments(args: Array<String>) {
    if (args.contains(KEY_HELP)) showMessage(message = HELP_MESSAGE, exitAfterMessage = true)
    val argumentError = "ERROR: Invalid argument name: Ensure define arguments => app-name={\"MyProject\"} or {\"My Project\"} package-name={com.sample.myproject}"
    when (args.size) {
        1 -> when {
            args.first().startsWith(KEY_APP_NAME) -> showErrorMessage("ERROR: No package name has been provided")
            args.first().startsWith(KEY_PACKAGE_NAME) -> showErrorMessage("ERROR: No app name has been provided")
            else -> showErrorMessage(argumentError)
        }
        ...
    }
}
```

Then `--help` is independent of the arguments size

Reply:
Hmm, I'm thinking should we show `HELP_MESSAGE` instead of `argumentError` 🤔

Reply:
@Wadeewee Hmmmmm, good point 😂 

What do you think @kaungkhantsoe?

Reply:
Or should we suggest the user run script `kscript new_project.kts --help` when the argument is an error instead 🤔

Reply:
Sorry guys, I totally forgot about this PR. I agree with @Wadeewee. But I would also like to inform user about the arguments were error too. So how about we combine two ideas? We will show the usual help message if it is run with `--help` option and if there was an error, we will append `ERROR: Invalid argument name:` to the help message. So that the user can know the error state and also learn how to make a correct one. @Tuubz @Wadeewee

Reply:
Works for me 👍